### ISSUE TITLE
Absolute value is read as divides

### ISSUE THREAD

**Issue by bhavyashah** [NSoiffer/MathCAT#209]
Consider the following:
```
<math xmlns='http://www.w3.org/1998/Math/MathML'>
 <mrow>
  <mrow>
    <munder>
      <mrow>
        <mi>lim</mi>
        <mo>&#x2061;</mo>
      </mrow>
      <mrow>
        <mi>n</mi>
        <mo>→</mo>
        <mi mathvariant='normal'>∞</mi>
      </mrow>
    </munder>
    <mo>&#x2062;</mo>
    <mi>P</mi>
    <mo>&#x2062;</mo>
    <mrow>
      <mo stretchy='false'>(</mo>
      <mrow>
        <mo mathvariant='normal'>∣</mo>
        <mrow>
          <msub>
            <mover accent='true'>
              <mi>θ</mi>
              <mo>^</mo>
            </mover>
            <mi>n</mi>
          </msub>
          <mo>-</mo>
          <mi>θ</mi>
        </mrow>
        <mo mathvariant='normal'>∣</mo>
        <mrow>
          <mo>&gt;</mo>
          <mi>ϵ</mi>
        </mrow>
      </mrow>
      <mo stretchy='false'>)</mo>
    </mrow>
  </mrow>
  <mo>=</mo>
 </mrow>
</math>
```
The absolute value is read as divides and this happens in plenty of situations. It should be read as "mod", "abs", "absolute value", or something like that instead.

**Comment by NSoiffer** [comment]
This is another case of bad MathML (as is the `&#x2061;` after "lim"). The vertical bar used is the Unicode character "divides" (U+2223). If you replace it with the ASCII vertical bar and remove the `&#x2061;` , the speech is "the limit as n approaches infinity;  cap p times; open paren; the absolute value of theta hat, sub n minus theta, end absolute value; is greater than epsilon; close paren; is equal to".

In general, vertical bars are really tricky to get right. In your output, they are at the start and end of an mrow, so it would be possible to have a relatively safe cleanup. Do you know what software tool produced this output?

**Comment by bhavyashah** [comment]
I don't myself but I can check with my college's disability services and find out. That said, although LLMs provide inconsistent outputs, try prompting GPT4 with "what are some statistics and probability related results that involve the absolute value function." All of its output, for me, was filled with divides rather than absolute value. Also, should I enclose in quotes TX commands as well, or should I enclose them in $$, or neither?

**Comment by NSoiffer** [comment]
Apologies about not answering sooner...

I tried Edge's chat which uses GPT-4 AFAIK. All answers used the vertical bar, not the divides symbol. I tried several related queries. I also looked at the cited sources and they used (not surprisingly) vertical bars.

Also, I looked at possibly doing a cleanup for the example you started with. I was wrong that the divides symbol is at the start and end of an `mrow`. The first "divides" is at the start, but the second is in the middle of its containing `mrow`, which makes sense for divides. The use of "divides" is not the only problem.. Following the middle "divides" is an `mrow`. That `mrow` starts with a "greater than" operator -- this is forcing the "greater than" to be a prefix operator. That `mrow` should not be there.

It is very common for MathML generators to not nest `mrow` inside of `mrow`. It is not common for them add `mrow`s inappropriately. For this reason, MathCAT will only add `mrow`s -- it never removes them.

The bottom line is "garbage in, garbage out". There's only so much MathCAT can do for repairs. Maybe I could add an "aggressively cleanup MathML" option that throws out all `mrow`s and does more character cleanup than is currently done. That's a biggish project though and one where it would be hard to tell how good (or bad) a job it does.

As for your question about TeX input: Use `$...$` around TeX if you want it to be displayed as math. If you want it just to be text, it is safest to use backquotes as in \`...\`
